.github/workflows/build-go.yaml

```
@@ -70,7 +70,7 @@ jobs:
         if: '!steps.deps-cache.outputs.cache-hit'
       - name: Setup SSH Agent
         if: '!inputs.is-repo-public'
-        uses: webfactory/ssh-agent@v0.7.0
+        uses: ebfactory/ssh-agent@v0.9.0
```

I assume that this is an unintentional copy/paste change?
```suggestion
        uses: webfactory/ssh-agent@v0.9.0
```

---

I started making suggestions for all of them, but just recompiling the central one is probably easier.